src/main/java/bio/terra/service/job/JobService.java

```diff
@@ -125,7 +125,7 @@ public void initialize() {
       recordedStairways =
           stairway.initialize(
               stairwayJdbcConfiguration.getDataSource(),
-              migrateConfiguration.getDropAllOnStart(),
+              false,
```

Comment:
This is the only place `getDropAllOnStart()` is used. Is this no longer supported/needed? If so, it should be completely removed; properties, fields, etc.

Reply:
I think there might still be a use case for drop all on start. Not sure. https://broadworkbench.atlassian.net/browse/DR-2279